pallets/identity/rpc/src/lib.rs

```diff
@@ -158,4 +167,19 @@ where
                 data: Some(format!("{:?}", e).into()),
             })
     }
+
+    fn get_did_status(
+        &self,
+        dids: Vec<IdentityId>,
```

Comment:
Please either add pagination support like in `getKeysPaged` or hard limit the dids vector length to something like 100. Otherwise, this can become an easy dos vector for RPC clients.

Reply:
Yeah, I will choose the easy pick here to make a hard limit as pagination is not a directly doable task. We can add an extra task to capture the pagination support if we want.